test/system/open_project/collapsible_section_test.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require "system/test_case"
+
+class IntegrationOpenProjectCollapsibleSectionTest < System::TestCase
+  def test_renders_component
+    visit_preview(:default)
+
+    assert_selector(".CollapsibleSection")
+  end
+
+  def test_renders_collapsed
+    visit_preview(:collapsed)
+
+    assert_selector(".octicon.octicon-chevron-up.d-none", visible: false)
+    assert_no_selector(".octicon.octicon-chevron-down.d-none")
```

Comment:
I think the following expresses intent a bit better:

```suggestion
    assert_selector(".octicon.octicon-chevron-up", visible: false)
    assert_selector(".octicon.octicon-chevron-down")
```

`d-none` is also a utility class*. I think we should aim to use semantic classes and attributes in our system test assertions. 

* we might want to discuss replacing `d-none` with the global `hidden` attribute. However if we do continue using `d-none`, we should also apply `aria-hidden` for semantics.